api/server.py

```
@@ -1,19 +1,22 @@
-from typing import Optional
-
-from fastapi import FastAPI, HTTPException
-from fastapi.responses import JSONResponse
+import time
+from hubspot import HubSpot
+from hubspot.crm.deals import SimplePublicObjectInput
 from pydantic import BaseModel
+from typing import Optional
 
-
+import os
```

Ruff (0.14.10)

Redefinition of unused `os` from line 3

Remove definition: `os`

(F811)